public/static/locales/en/form-builder.json

```diff
@@ -363,5 +363,12 @@
   "dyanamicRow": "Repeating question",
   "addToSet": "Add to set",
   "removeFromSet": "Remove from set",
-  "untitledForm": "Unnamed form file"
+  "untitledForm": "Unnamed form file",
```

Comment:
```suggestion
  "untitledForm": "Untitled form",
```

Reply:
I'll make a separate PR to update `untitledForm` for be unNamed form

The "filename" can be separate from the form title.  "Filename" is what gets displayed under the "My Forms" section

<img width="721" alt="Screen Shot 2023-02-17 at 8 15 33 AM" src="https://user-images.githubusercontent.com/62242/219662700-29a24542-7269-4ced-9867-c39493d43bed.png">